Exercise_4/docs/part-3.md

```diff
@@ -1 +1,204 @@
+
+The result of this, however, is that the potential exists for two health IT systems, **even when both are on the FHIR® standard, to have vastly divergent sets of information available.** Because of this, there exists a need for FHIR® standard guidelines that define the precise implementations of FHIR® a specific health IT system has deployed and supports. These guides have been formalized in FHIR® as **Implementations Guides (IG)**.
+
+>An IG is a set of rules about how FHIR® resources are used (or should be used) to solve a particular problem, with associated documentation to support and clarify the usage.
+
+FHIR® IGs are typically published on the web after they are generated.
+
+??? example "For more information on the general process for IG creation, continue here"
+    * FHIR® has a specific process for publishing IG available here: [https://confluence.hl7.org/display/FHIR/IG+Publisher+Documentation](https://confluence.hl7.org/display/FHIR/IG+Publisher+Documentation){target=_blank}
+
+    * FHIR® also includes the `ImplementationGuide` resource: a single resource that defines the logical structure of the content of the IG, so that the contents of the logical package that the IG represents are computable. 
+
+    * In particular, validators are able to use the `CapabilityStatement` resource to validate content against the IG as a whole. The capability statement is a key part of the overall conformance framework in FHIR. It is used as a statement of the features of actual software, or of a set of rules for an application to provide. This statement connects to all the detailed statements of functionality, such as StructureDefinitions and ValueSets. This composite statement of application capability may be used for system compatibility testing, code generation, or as the basis for a conformance assessment. For further information about Conformance testing, see Conformance Rules and Profiling FHIR.
+    More on this resource here: [https://build.fhir.org/capabilitystatement.html](https://build.fhir.org/capabilitystatement.html) 
+
+    * FHIR® contains an official registry of IG available here: [http://registry.fhir.org/](http://registry.fhir.org/)
+
+It should also be noted that while individual organizations and entities can develop their own IG, the development process should also be guided and supported by the HL7 community.
+
+## US Core 
+So, if there are a variety of IGs available for FHIR® implementation, is there a particular one that maps to the USCDI data standard, one whose adoption would ensure that an EHR system fully met Promoting Interoperability guidelines?
+
+> Yes! The US Core IG does exactly that.
+
+The US Core IG is based on FHIR® Version R4 
```

Comment:
```suggestion
The current, as of writing, version of the US Core IG (5.0.0) is based on FHIR® Version R4.
```

It's important to clarify the US Core version here since earlier versions were based on different versions of FHIR and in the future this may change again.